templates/cpp-template-default/proj.android/app/build.gradle

```diff
@@ -111,19 +111,22 @@ android {
 }
 
 android.applicationVariants.all { variant ->
-    def project_root_folder = "${buildDir}/../../.."
-    def dest_assets_folder = "${buildDir}/../assets"
+    def project_root_folder = "${projectDir}/../.."
+    def dest_assets_folder = "${projectDir}/assets"
 
     // delete previous files first
     delete dest_assets_folder
-
-    variant.mergeAssets.doFirst {
+    def targetName = variant.name.capitalize()
+    def copyTaskName = "copy${targetName}ResourcesToAssets"
+    
+    tasks.register(copyTaskName) {
         copy {
             from project_root_folder + "/Resources"
             into dest_assets_folder
             exclude "**/*.gz"
         }
     }
+    tasks.getByName("pre${targetName}Build").dependsOn copyTaskName
```

Comment:
What does it mean?

Reply:
To make task `copyDebug(Release)ResourcesToAsset` run before `preDebug(Release)Build`, which will always run ahead of build.